.env.example

```diff
@@ -51,6 +52,31 @@ BUNDLE_TRACKER_ADDRESS=0xF96cd74e7EEcb93a773105269b7ef5187db30aef
 PROPOSER_ADDRESS=0xYourPublicKeyHere
 PROPOSER_KEY=0xYourPrivateKeyHere
 
+# ─────────────────────────────────────────────────────────────────────────────
+# WEBHOOK CONFIGURATION
+# ─────────────────────────────────────────────────────────────────────────────
+#
+# Used when the node (bundle proposer) sends notifications to the web app
+# whenever a bundle is proposed or finalized.
+#
+
+# Webhook endpoint URL on the web app server
+# Example: https://your-webapp.example.com/api/protocol/webhook
+WEBHOOK_URL=https://your-webapp.example.com/api/protocol/webhook
+
+# Shared secret for authenticating node → web app webhook requests
+# Keep this secret! Never expose it publicly.
+# Generate with: node -e "console.log(require('crypto').randomBytes(32).toString('hex'))"
+WEBHOOK_SECRET=your-generated-webhook-secret
```

Comment:
Would we want a collection of webhooks for broadcasting to many different endpoints (see above)?